version/version.go

```diff
@@ -17,7 +17,7 @@ package version
 
 var (
 	// Version is the operator version
-	Version = "v0.11.0+dev"
+	Version = "v0.12.0-dev"
```

Comment:
hmm, this seemed to change from a `+` to a `-`

Reply:
Yeah I set it up to do it this way because semver specifies
```
<valid semver> ::= <version core>
                 | <version core> "-" <pre-release>
                 | <version core> "+" <build>
                 | <version core> "-" <pre-release> "+" <build>
```
i.e. it looks like `+` is for build identifiers, `-` is for pre-release. We could stick with `+dev` but from my reading that's meant for e.g. a commit hash or other identifier.